sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,48 @@
+---
+parent: 'test_cases/test_branch_origin_service_catalog.yml'
+overrides:
+  sync:
+    - "openshift,ansible-service-broker=master"
+  junit_analysis: False
+  timer: 'H H * * *'
+  email:
+    - ernelson@redhat.com
+extensions:
+  sync_repos:
+    - "ansible-service-broker"
+  actions:
+    - type: "script"
+      title: "install asb to cluster"
+      repository: "ansible-service-broker"
+      timeout: 1800
+      script: |-
+        $GOPATH/src/github.com/origin/_output/local/bin/linux/amd64/oc create project ansible-service-broker
+        $GOPATH/src/github.com/origin/_output/local/bin/linux/amd64/oc process ./templates/deploy-ansible-service-broker.template.yaml -p BROKER_CA_CERT=$(oc get secret -n kube-service-catalog -o go-template=\'{{ range .items }}{{ if eq .type \"kubernetes.io/service-account-token\" }}{{ index .data \"service-ca.crt\" }}{{end}}{{\"\\n\"}}{{end}}\' | tail -n 1) | oc create -n ansible-service-broker -f -'
```

Comment:
If oc is indeed not in the path, you'll need to specify the full path throughout the command.